packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -223,6 +234,62 @@ def test_containers(app_endpoints):
         test_containers(endpoints)
 
 
+def test_metrics_containers_app(dcos_api_session):
+    """Assert that app metrics appear in the v0 metrics API."""
+    task_name = 'test-metrics-containers-app'
+    metric_name_pfx = 'test_metrics_containers_app'
+    marathon_app = {
+        'id': '/' + task_name,
+        'instances': 1,
+        'cpus': 0.1,
+        'mem': 128,
+        'cmd': '\n'.join([
+            'echo "Sending metrics to $STATSD_UDP_HOST:$STATSD_UDP_PORT"',
+            'echo "Sending gauge"',
+            'echo "{}.gauge:100|g" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+
+            'echo "Sending counts"',
+            'echo "{}.count:1|c" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+            'echo "{}.count:1|c" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+
+            'echo "Sending timings"',
+            'echo "{}.timing:1|ms" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+            'echo "{}.timing:2|ms" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+            'echo "{}.timing:3|ms" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+
+            'echo "Sending histograms"',
+            'echo "{}.histogram:1|h" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+            'echo "{}.histogram:2|h" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+            'echo "{}.histogram:3|h" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+            'echo "{}.histogram:4|h" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT'.format(metric_name_pfx),
+
+            'echo "Done. Sleeping forever."',
+            'while true; do',
+            '  sleep 1000',
+            'done',
+        ]),
```

Comment:
Just a readability comment. I think inlining `metric_name_pfx` will be more readable on the other hand whole `nc` part could be extracted to constatn and injected with format
```bash
echo "Sending metrics to $STATSD_UDP_HOST:$STATSD_UDP_PORT"
echo "Sending gauge"
echo "test_metrics_containers_app.gauge:100|g" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "Sending counts"
echo "test_metrics_containers_app.count:1|c" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "test_metrics_containers_app.count:1|c" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "Sending timings"
echo "test_metrics_containers_app.timing:1|ms" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "test_metrics_containers_app.timing:2|ms" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "test_metrics_containers_app.timing:3|ms" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "Sending histograms"
echo "test_metrics_containers_app.histogram:1|h" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "test_metrics_containers_app.histogram:2|h" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "test_metrics_containers_app.histogram:3|h" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "test_metrics_containers_app.histogram:4|h" | nc -w 1 -u $STATSD_UDP_HOST $STATSD_UDP_PORT
echo "Done. Sleeping forever."
while true; do
sleep 1000
done
```